packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -2474,39 +3016,49 @@ class PaddleSliderValueIndicatorShape extends SliderComponentShape {
     assert(labelPainter != null);
     assert(parentBox != null);
     assert(sliderTheme != null);
+    assert(!sizeWithOverflow.isEmpty);
     final ColorTween enableColor = ColorTween(
       begin: sliderTheme.disabledThumbColor,
       end: sliderTheme.valueIndicatorColor,
     );
-    _pathPainter.drawValueIndicator(
+    _pathPainter.paint(
       parentBox,
       context.canvas,
       center,
       Paint()..color = enableColor.evaluate(enableAnimation),
       activationAnimation.value,
       labelPainter,
+      textScaleFactor,
+      sizeWithOverflow,
       null,
     );
   }
 }
 
-/// The default shape of a [RangeSlider]'s value indicators.
+/// A variant shape of a [RangeSlider]'s value indicators.
```

Comment:
Same comments as for the slider version

Reply:
The reason it is switched to variant is because there are multiple value indicators now. Afterwards this value indicator will no longer by the default. That is why the phrase has switched slightly.

Reply:
https://github.com/flutter/flutter/pull/52663#discussion_r399543532

Reply:
Updated